src/Tests/Framework/TestClient.cs

```diff
@@ -97,28 +101,33 @@ private static IClrTypeMapping<Project> ProjectMapping(ClrTypeMappingDescriptor<
 			bool forceInMemory = false,
 			Func<Uri, IConnectionPool> createPool = null,
 			Func<ConnectionSettings, IElasticsearchSerializer> serializerFactory = null
-			)
+		)
 		{
 			createPool = createPool ?? (u => new SingleNodeConnectionPool(u));
 #pragma warning disable CS0618 // Type or member is obsolete
-			var defaultSettings = DefaultSettings(new ConnectionSettings(createPool(CreateUri(port)), CreateConnection(forceInMemory: forceInMemory), serializerFactory));
+			var defaultSettings = DefaultSettings(new ConnectionSettings(createPool(CreateUri(port)),
+				CreateConnection(forceInMemory: forceInMemory), serializerFactory));
 #pragma warning restore CS0618 // Type or member is obsolete
 			var settings = modifySettings != null ? modifySettings(defaultSettings) : defaultSettings;
 			return settings;
 		}
 
-		public static IElasticClient GetInMemoryClient(Func<ConnectionSettings, ConnectionSettings> modifySettings = null, int port = 9200) =>
+		public static IElasticClient GetInMemoryClient(Func<ConnectionSettings, ConnectionSettings> modifySettings = null,
```

Comment:
Keep as one line? we should settle on a line length, general code formatting etcetera.